software/controller/test/sensor_tests/sensors_test.cpp

```diff
@@ -54,23 +55,33 @@ static Voltage MPXV5004_PressureToVoltage(Pressure pressure) {
   return volts(3.3f * (0.2f * pressure.kPa() + 0.2f));
 }
 
+// This method models the oxygen sensor. Voltage reads partial pressure of O2
+// and needs ambient pressure to modulate output voltage.
+// Sensitivity is 0.061 V/pt at 1 atm and assumed to vary linearly with pressure
+// Offset is assumed to be 10 mV
+// Analog amplifier with gain 50 is used on the PCB
```

Comment:
Any possible link to literature?

Reply:
I really wish oxygen sensor datasheets were better about this.  It's basically just assumed in most cases that the sensitivity is this value for all 13mV ambient sensors.  It might be better to calibrate the sensor gain as well as the offset.  However, this requires a reliable source of known-concentration oxygen.  If the hospital's supply is 95 % oxygen (such as that from any oxygen concentrator), then this will be inaccurate.

Reply:
I don't have a good understanding of what the standard here is for medical devices, and what they can assume about the supply gas.

Reply:
resolved and committed change, using https://www.apogeeinstruments.com/content/SO-100-200-spec-sheet.pdf as the source doc for sensitivity.